src/components/HeroImage/HeroImage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { s } from "./";
+
+export default function HeroImage(props){
+
+	const {
+		image: {
+			src,
+			mask: maskPath,
+			description
+		}
+	} = props;
+
+	// inline style to prevent the need for a huge png asset
+	const mask = { clipPath: `url(${maskPath})` };
+
+	return (
+		<div className={ s.hero } id="hero"> 	
```

Comment:
```suggestion
		<div className={ s.wrapper} id="character__hero__ragnar_lothbrok"> 	
```
'hero' is a bit generic; we might have other hero images; let's namespace to `character__hero` and also create a `__character_name` identifer on a per-image basis.